signatures/LUOV/LUOV/bench/avx2-chacha8_LUOV-8-107-371_asus.yaml

```diff
@@ -0,0 +1,8 @@
+platform: ASUS S410 Notebook PC with an Intel Core i5-8250U CPU @ 1.60 GHz
+timings:
+  measurements:
+    gen: 9600000
+    enc|sign: 3100000
+    dec|vrfy: 1500000
+  unit: cycles
+  comment: average of 1000 executions compiled with ”gcc − O3” with gcc version 7.3.0
```

Comment:
(@all files in bench/) Not the most important thing but there's a typographic whatever dash not the ascii/shell minus one. I wonder how that happened, do you have you editor set to magicness=high or something? :D